Dockerfile

```diff
@@ -4,6 +4,9 @@ MAINTAINER Zalando SE
 
 COPY target/essentials.jar /
 
-CMD java $(java-dynamic-memory-opts) $(appdynamics-agent) -jar /essentials.jar
+EXPOSE 8080
+ENV HTTP_PORT=8080
```

Comment:
To be honest: I don't know, but it's also in our other Dockerfiles. I figured it has a purpose.